source/patterns/00-atoms/doi/doi.twig

```diff
@@ -0,0 +1,11 @@
+{%- import 'atoms-html' as html -%}
+
+{%- with {doi: doi|default(null)} only -%}
+  {%- block doi -%}
+
+    {%- if doi -%}
+      {%- include 'atoms-link' with {attributes: {class: 'doi', href: 'https://doi.org/'~doi}, text: 'https://doi.org/'~doi} only -%}
```

Comment:
Inside `contextual-data` (see previous comment), this should be amended to `class: 'contextual-data__doi'`